internal/cmd/beta/security-group/create/create.go

```diff
@@ -0,0 +1,171 @@
+
+			if err := outputResult(p, model, group); err != nil {
+				return err
+			}
+
+			return nil
+		},
+	}
+
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().String(nameFlag, "", "The name of the security group.")
+	cmd.Flags().String(descriptionFlag, "", "An optional description of the security group.")
+	cmd.Flags().Bool(statefulFlag, false, "Create a stateful or a stateless security group")
+	cmd.Flags().StringToString(labelsFlag, nil, "Labels are key-value string pairs which can be attached to a network-interface. E.g. '--labels key1=value1,key2=value2,...'")
+
+	if err := flags.MarkFlagsRequired(cmd, nameFlag); err != nil {
+		cobra.CheckErr(err)
+	}
+}
+
+func parseInput(p *print.Printer, cmd *cobra.Command) (*inputModel, error) {
+	globalFlags := globalflags.Parse(p, cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+	name := flags.FlagToStringValue(p, cmd, nameFlag)
+
+	model := inputModel{
+		GlobalFlagModel: globalFlags,
+		Name:            &name,
+
+		Labels:      flags.FlagToStringToStringPointer(p, cmd, labelsFlag),
+		Description: flags.FlagToStringPointer(p, cmd, descriptionFlag),
+		Stateful:    flags.FlagToBoolPointer(p, cmd, statefulFlag),
+	}
+
+	if p.IsVerbosityDebug() {
+		modelStr, err := print.BuildDebugStrFromInputModel(model)
+		if err != nil {
+			p.Debug(print.ErrorLevel, "convert model to string for debugging: %v", err)
+		} else {
+			p.Debug(print.DebugLevel, "parsed input values: %s", modelStr)
+		}
+	}
+
+	return &model, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *iaas.APIClient) iaas.ApiCreateSecurityGroupRequest {
+	request := apiClient.CreateSecurityGroup(ctx, model.ProjectId)
+
+	var labelsMap *map[string]any
+	if model.Labels != nil && len(*model.Labels) > 0 {
+		// convert map[string]string to map[string]interface{}
+		labelsMap = utils.Ptr(map[string]interface{}{})
+		for k, v := range *model.Labels {
+			(*labelsMap)[k] = v
+		}
+	}
+	payload := iaas.CreateSecurityGroupPayload{
+		Description: model.Description,
+		Labels:      labelsMap,
+		Name:        model.Name,
+		Stateful:    model.Stateful,
+	}
+
+	return request.CreateSecurityGroupPayload(payload)
+}
+
+func outputResult(p *print.Printer, model *inputModel, resp *iaas.SecurityGroup) error {
+	switch model.OutputFormat {
+	case print.JSONOutputFormat:
+		details, err := json.MarshalIndent(resp, "", "  ")
+		if err != nil {
+			return fmt.Errorf("marshal security group: %w", err)
+		}
+		p.Outputln(string(details))
+
+		return nil
+	case print.YAMLOutputFormat:
+		details, err := yaml.MarshalWithOptions(resp, yaml.IndentSequence(true))
+		if err != nil {
+			return fmt.Errorf("marshal security group: %w", err)
+		}
+		p.Outputln(string(details))
+
+		return nil
+	default:
+		operationState := "Created"
+		p.Outputf("%s security group %q\n", operationState, *model.Name)
```

Comment:
This can be simplified to:

```suggestion
		p.Outputf("Created security group %q\n", *model.Name)
```